scylla-cql/src/frame/response/result.rs

```diff
@@ -1269,109 +1342,116 @@ pub fn deser_cql_value(
     let v = Some(FrameSlice::new_borrowed(buf));
```

Comment:
:pushpin: When removing the legacy deserialization framework, please check if it's possible to remove the workaround in `FrameSlice`: the `new_borrowed` constructor.